scripts/scil_fibertube_compute_density.py

```diff
@@ -0,0 +1,161 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Estimates the per-voxel volumetric density of a set of fibertubes. In other
+words, how much space is occupied by fibertubes and how much is emptiness.
+
+1. Segments voxels that contain at least a single fibertube.
+2. Valid voxels are finely sampled and we count the number of samples that
+landed within a fibertube. For each voxel, this number is then divided by
+its total amount of samples.
+3. By doing the same steps for samples that landed within 2 or more
+fibertubes, we can create a density map of the fibertube collisions.
+
+See also:
+    - docs/source/documentation/fibertube_tracking.rst
+"""
+
+import os
+import json
+import nibabel as nib
+import argparse
+import logging
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram
+from scilpy.tractanalysis.fibertube_scoring import fibertube_density
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg,
+                             add_verbose_arg,
+                             add_json_args)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter,
+        description=__doc__)
+
+    p.add_argument('in_fibertubes',
+                   help='Path to the tractogram (must be .trk) file \n'
+                        'containing fibertubes. They must be: \n'
+                        '1- Void of any collision. \n'
```

Comment:
1 - void of any collision: why does it matter?

Reply:
No it doesn't matter. I think it was just to stay consistent with previous documentation. It's "technically" impossible to have fibertubes without filtering. I'll try to do something about it.